administrator/components/com_admin/sql/updates/postgresql/4.0.0-2020-12-31.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM `#__postinstall_messages` WHERE `title_key` = 'TPL_HATHOR_MESSAGE_POSTINSTALL_TITLE';
+DELETE FROM `#__postinstall_messages` WHERE `title_key` = 'PLG_PLG_RECAPTCHA_VERSION_1_POSTINSTALL_TITLE';
+DELETE FROM `#__postinstall_messages` WHERE `title_key` = 'COM_CPANEL_MSG_JOOMLA40_PRE_CHECKS_TITLE';
```

Comment:
```suggestion
DELETE FROM "#__postinstall_messages" WHERE "title_key" = 'TPL_HATHOR_MESSAGE_POSTINSTALL_TITLE';
DELETE FROM "#__postinstall_messages" WHERE "title_key" = 'PLG_PLG_RECAPTCHA_VERSION_1_POSTINSTALL_TITLE';
DELETE FROM "#__postinstall_messages" WHERE "title_key" = 'COM_CPANEL_MSG_JOOMLA40_PRE_CHECKS_TITLE';
```
Use the right names quoting for PostgreSQL.